### DISCUSSION TITLE
Export with adiccional Customer fields

### DISCUSSION THREAD

**Discussion by uribes78** [kimai/kimai#5065] [Questions & Answers]
Using the default PDF export, we need to add the `contact` and `description` fields into the summary section, I have tried modifying Project entity because I saw this: 

```php
#[Exporter\Order(['id', 'name', 'customer', 'orderNumber', 'orderDate', 'start', 'end', 'budget', 'timeBudget', 'budgetType', 'color', 'visible', 'teams', 'comment', 'billable', 'number'])]
#[Exporter\Expose(name: 'customer', label: 'customer', exp: 'object.getCustomer() === null ? null : object.getCustomer().getName()')]
#[Constraints\Project]
class Project implements EntityWithMetaFields, EntityWithBudget
{
``` 
Thinking that `Export\Expose` allow to get the `customer` name, I tried adding the same but for `contact` field, it did'nt failed once a clear and create cache, but is showing blank when I print the contact field. I tried this:

On Project entity add this:

`#[Exporter\Expose(name: 'contact', label: 'contact', exp: 'object.getCustomer().getCustomer() === null ? null : object.getCustomer().getContact()')]`

and on pdf twig I tried this:

```html
            <tr class="{{ cycle(['odd', 'even'], customerCount) }}">
                {% block summary_project_row_start %}{% endblock %}
                {% for summaryColumn in summaryColumns %}
                    {% if summaryColumn == 'customer' %}
                        <td>{{ summary.customer }} - {{summary.contact}}</td>
                    {% elseif summaryColumn == 'project' %}
```

The question is: How can I add more fields from entities (Customer is this case) to be printed on the PDF export? Thanks for advice.

I'm using Kimai v2, PHP 8.2. Thanks

**Comment by uribes78** [comment]
@kevinpapst , What's the price to get support to have my report as expected? I need to know how to show 'Contact' field and 'Description' on summary section at Export Standart PDF? Please your advice! Thanks

**Reply by kevinpapst** [reply]
This discussion section is for the open source version of Kimai. Contact infos for commercial support are listed on the website.
